pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -174,6 +168,13 @@ let
         license = licenses.cddl;
         platforms = platforms.linux;
         maintainers = with maintainers; [ hmenke jcumming jonringer wizeman fpletz globin mic92 ];
+        broken = if
+          buildKernel && (incompatibleKernelVersion != null) && versionAtLeast kernel.version incompatibleKernelVersion
```

Comment:
```suggestion
          buildKernel && (incompatibleKernelVersion != null) && kernel.kernelAtLeast incompatibleKernelVersion
```

Reply:
LGTM, feel free to just push that.

Could you also do the backport to stable (staging?)? You'd probably be the one to merge my PR for that anyways ;)

Reply:
Although hold on, have we foreward-ported kernelAtLeast yet? You created that in staging IIRC.

Reply:
I should be there, I needed it to properly mark failing module builds

Reply:
```
[11:42:55] jon@jon-desktop /home/jon/projects/nixpkgs (release-20.09)
$ git pull
Already up to date.
[11:42:59] jon@jon-desktop /home/jon/projects/nixpkgs (release-20.09)
$ nix eval -f . linuxPackages.kernel.kernelAtLeast
<LAMBDA>
```